INTITA/Modules/TasksScreen/TasksViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  TasksViewController.swift
+//  INTITA
+//
+//  Created by Svitlana Korostelova on 25.12.2020.
+//
+
+import UIKit
+
+class TasksViewController: UIViewController, Storyboarded, AlertAcceptable, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, UIScrollViewDelegate {
+    
+    var viewModel: TasksViewModel!
+    weak var coordinator: TasksCoordinator?
+    @IBOutlet weak var tasksHeadersCollectionView: UICollectionView!
+    @IBOutlet weak var taskTableView: UITableView!
+    @IBOutlet weak var priorities: UIButton!
+    @IBOutlet weak var types: UIButton!
+    @IBOutlet weak var groups: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.setNavigationBarHidden(false, animated: true)
+        let backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: self, action: #selector(backButtonPressed))
+                self.navigationController?.navigationBar.backItem?.backBarButtonItem = backBarButtonItem
+        //TODO:  set title for controller
+        let headerCell = UINib(nibName: "TaskHeaderCell", bundle: nil)
+        tasksHeadersCollectionView.register(headerCell, forCellWithReuseIdentifier: "HeaderReuseIdentifier")
+        let taskCell = UINib(nibName: "TaskCell", bundle: nil)
+        taskTableView.register(taskCell, forCellReuseIdentifier: "reuseForTask")
+        tasksHeadersCollectionView.bordered()
+        tasksHeadersCollectionView.rounded()
+        taskTableView.bordered()
+        taskTableView.rounded()
+        priorities.titleLabel?.font = UIFont.primaryFontLight
+        priorities.setTitle("priorities".localized, for: .normal)
+        types.titleLabel?.font = UIFont.primaryFontLight
+        types.setTitle("type".localized, for: .normal)
+        groups.titleLabel?.font = UIFont.primaryFontLight
+        groups.setTitle("groups".localized, for: .normal)
+    }
+    @objc func backButtonPressed(){
+        coordinator?.showProfileScreen()
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return viewModel.numberOfStates
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let state = State(rawValue: indexPath.row),
+              let sectionHeader = collectionView.dequeueReusableCell(withReuseIdentifier: "HeaderReuseIdentifier", for: indexPath) as? TaskHeaderCell else { return UICollectionViewCell() }
```

Comment:
collectionView.dequeueReusableCell(withReuseIdentifier: "HeaderReuseIdentifier", for: indexPath) -> collectionView.dequeueReusableCell(withReuseIdentifier: TaskHeaderCell.identifire, for: indexPath)